ui/app/components/http-requests-table.js

```diff
@@ -0,0 +1,51 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+/**
+ * @module HttpRequestsTable
+ * HttpRequestsTable components render a table with the total number of HTTP Requests to a Vault server per month.
+ *
+ * @example
+ * ```js
+ * const COUNTERS = [
+ *    {
+ *       "start_time": "2019-05-01T00:00:00Z",
+ *       "total": 50
+ *     }
+ * ]
+ *
+ * <HttpRequestsTable @counters={{COUNTERS}} />
+ * ```
+ *
+ * @param counters {Array} - A list of objects containing the total number of HTTP Requests for each month. `counters` should be the response from the `/internal/counters/requests` endpoint.
+ */
+
+export default Component.extend({
+  classNames: ['http-requests-table'],
+  counters: null,
+  showChangeColumn: computed('counters', function() {
+    const { counters } = this;
+    return counters && counters.length > 1;
+  }),
+  countersWithChange: computed('counters', function() {
+    const { counters } = this;
+
+    if (counters) {
```

Comment:
is there a better way to ensure this value is only calculated once counters is no longer `null`? it seems odd to have to defensively check if `counters` exists both here and in `showChangeColumn` above.

Reply:
The computed property will cache, but will recompute when counters changes value, so you have to account for both cases. You could do `let counters = this.counters || []` above so you could drop this `if` if you wanted.  For the `showChangeColumn`, I'd just move that to the template as `{{#if (gt counters.length 1)}}` (using the helper from here: https://github.com/jmurphyau/ember-truth-helpers#usage).